src/rules/no-flowmax-in-forwardref.coffee

```diff
@@ -0,0 +1,23 @@
+module.exports =
+  meta:
+    docs:
+      description: 'Flag uses of flowMax() within forwardRef()'
+      category: 'Possible Errors'
+      recommended: yes
+    schema: [
+      type: 'object'
```

Comment:
I *believe* that for a rule that doesn't accept any options, you'd write that as `schema: []`

I think that what you have here is saying "it can accept an empty object option" eg
```
rules: {
  'ad-hok/no-flowmax-in-forwardref': ['error', {}]
```
(which is unnecessary to support)